src/workers/notifications/postAdded.ts

```diff
@@ -66,6 +74,18 @@ const worker: NotificationWorker = {
             } as NotificationPostContext & Partial<NotificationDoneByContext>,
           }),
         );
+
+        const posts = await con.getRepository(Post).countBy({
+          authorId: post.authorId,
+        });
+
+        if (posts === 1) {
```

Comment:
This would just be a "Extra"? 
Because technically they could delete a post, then post again (I do know ignore will trigger)

Reply:
Oh, but when they delete a post we just set the property `deleted = true` right?

Reply:
Indeed. Should be available on the condition now 🥳

Reply:
I think we discussed in the other PR that we don't need to count the posts if we add retroactively the actions

Reply:
Yes, I will rebase this once the PR is merged.